dynawo/sources/Modeler/Common/DYNCompiler.cpp

```diff
@@ -175,8 +175,8 @@ Compiler::getDDB() {
     // scan for all .mo files (for sanity checks purposes)
     searchModelsFiles(DDBDir, ".mo", noFileExtensionsForbidden, searchInSubDirsStandardModels, packageNeedsRecursive, !stopWhenSeePackage, moFilesAll_);
 
-    // scan for all .xml (external variables files)
-    searchModelsFiles(DDBDir, ".xml", fileExtensionsForbiddenXML, searchInSubDirsStandardModels, packageNeedsRecursive, !stopWhenSeePackage, extVarFiles_);
+    // scan for all .extvar (external variables files)
+    searchModelsFiles(DDBDir, ".extvar", fileExtensionsForbiddenXML, searchInSubDirsStandardModels, packageNeedsRecursive, !stopWhenSeePackage, extVarFiles_);
```

Comment:
It means that the retrocompatibility with xml files is broken. MIght be better to keep it.